packages/agent-toolkit/package.json

```diff
@@ -39,7 +39,7 @@
     "build": "tsup --env.NODE_ENV production",
     "clean": "rimraf ./dist",
     "dev": "tsup --watch",
-    "lint": "eslint src",
+    "lint": "eslint --quiet src",
```

Comment:
This is kind of a big change, no? This suppresses ESLint warnings, and only reports errors. Is that necessary for this PR since each assertion has a `disable-next-line`?

Reply:
Removed the flag in [6f06c14](https://github.com/clerk/javascript/pull/5038/commits/6f06c1471920cd1f568346d4e8b3636393de44a7)